core/src/main/java/org/stellar/anchor/apiclient/PlatformApiClient.java

```diff
@@ -13,13 +19,28 @@
 import org.springframework.data.domain.Sort;
 import org.stellar.anchor.api.exception.AnchorException;
 import org.stellar.anchor.api.exception.InvalidConfigException;
-import org.stellar.anchor.api.platform.*;
+import org.stellar.anchor.api.platform.GetTransactionResponse;
+import org.stellar.anchor.api.platform.GetTransactionsResponse;
+import org.stellar.anchor.api.platform.PatchTransactionsRequest;
+import org.stellar.anchor.api.platform.PatchTransactionsResponse;
+import org.stellar.anchor.api.rpc.RpcRequest;
+import org.stellar.anchor.api.rpc.method.AmountAssetRequest;
+import org.stellar.anchor.api.rpc.method.AmountRequest;
+import org.stellar.anchor.api.rpc.method.NotifyOnchainFundsReceivedRequest;
+import org.stellar.anchor.api.rpc.method.NotifyOnchainFundsSentRequest;
+import org.stellar.anchor.api.rpc.method.NotifyRefundSentRequest;
+import org.stellar.anchor.api.rpc.method.NotifyRefundSentRequest.Refund;
+import org.stellar.anchor.api.rpc.method.NotifyTransactionErrorRequest;
+import org.stellar.anchor.api.rpc.method.RpcMethod;
 import org.stellar.anchor.api.sep.SepTransactionStatus;
 import org.stellar.anchor.auth.AuthHelper;
 import org.stellar.anchor.util.OkHttpUtil;
 
 /** The client for the PlatformAPI endpoints. */
 public class PlatformApiClient extends BaseApiClient {
+
+  public static final String JSON_RPC_VERSION = "2.0";
```

Comment:
Is there a reason not to start from version `1.0`?

Reply:
Its the JSON RPC specification version: https://www.jsonrpc.org/specification#examples